src/mbgl/map/transform.cpp

```diff
@@ -517,7 +527,7 @@ double Transform::getAngle() const {
 #pragma mark - Pitch
 
 void Transform::setPitch(double pitch, const Duration& duration) {
-    setPitch(pitch, {NAN, NAN}, duration);
+    setPitch(pitch, ScreenCoordinate {}, duration);
```

Comment:
So if you don’t specify an `anchor` for a `setPitch()` call, we assume the gesture is anchored in the top-left corner of the view?